### PR TITLE
Fix typo in .json file name

### PR DESCRIPTION
**Pull request by sshantel**
Signed-off-by: Chantel Yip <52993239+sshantel@users.noreply.github.com>

* [ ] I added CHANGELOG entry for this change.
* [ x ] Change is not relevant to the end user.

### CODE REVIEW

File: docs/storage.md [hunk 1]
```diff
@@ -644,7 +644,7 @@ total 2209344
 drwxr-xr-x 2 bwplotka bwplotka       4096 Dec 10  2019 chunks
 -rw-r--r-- 1 bwplotka bwplotka 1962383742 Dec 10  2019 index
 -rw-r--r-- 1 bwplotka bwplotka       6761 Dec 10  2019 meta.json
--rw-r--r-- 1 bwplotka bwplotka        111 Dec 10  2019 delete-mark.json      # <-- Optional marker.
+-rw-r--r-- 1 bwplotka bwplotka        111 Dec 10  2019 deletion-mark.json      # <-- Optional marker.
```
Comment (yeya24) [inline docs/storage.md:647]: Can you format the `#` be the same indent as the next line? Thanks!

Comment (sshantel) [reply docs/storage.md]: ✅ Yes, thanks Ben!

**Review by fpetkovski — DISMISSED** [review]

**Comment by fpetkovski** [comment]
@sshantel we will need you to sign the DCO. You can do it with 
```
git commit --amend --no-edit --signoff
```
followed by a force push.

**Comment by sshantel** [comment]
@fpetkovski Thanks, done! I missed that part [here](https://thanos.io/tip/contributing/contributing.md/#pull-request-process) :)

**Comment by yeya24** [comment]
Oops sorry @sshantel. I merged this with main but seems it hit the DCO issue again. Would you mind rebasing main locally and force push? Then we should be good to merge, thanks!

**Comment by sshantel** [comment]
new PR here due to merge conflicts: https://github.com/thanos-io/thanos/pull/6087

**Pull request closed without merging**